test/integration_test/specs/mysql-localsnap-group/mysql.yaml

```diff
@@ -0,0 +1,77 @@
+kind: PersistentVolumeClaim
```

Comment:
Can you use the specs under `mysql-2-pvc` instead? You can add the labels to the PVCs there

Reply:
changed

Reply:
My bad, I thought you wanted to have different pvc spec for group snapshot restore.  I kept this as separate since we need specific `groupsnapshotname` in CRD and easy to read later for manual testing